examples/src/python/example/3rdparty_py.md

```diff
@@ -69,3 +69,27 @@ Then in your Python code, you can `import` from that package:
 
     :::python
     from colors import green
+
+Managing dependencies for multiple platforms
+----------------------
+
+If you're building a python binary for use on multiple platforms, you might have 3rd-party
+dependencies that rely on platform-specific code. In addition to specifying the platforms
+with which your binary is intended to be compatible in the `platform` field of your
+<a pantsref="bdict_python_binary">`python_binary`</a> target, you may need to make
+<a href="https://pip.pypa.io/en/stable/reference/pip_wheel/">wheel</a> files for each platform
+available at build time.
+
+You can store these files on the web formatted in a link detection-friendly format or locally in
```

Comment:
It feels like this paragraph needs a bit more intro. I think it would be helpful to introduce the idea of a repo before explaining the two kinds of repos.